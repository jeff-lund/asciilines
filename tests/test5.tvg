10 12
C 1 1 h 4
C 1 1 v 8
C 8 1 h 4
S 1 7 h 4
S 1 7 v 4
S 5 7 h 4
S 5 10 v 4
S 8 7 h 4 
