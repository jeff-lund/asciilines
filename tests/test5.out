............
.CCCC..SSSS.
.C.....S....
.C.....S....
.C.....S....
.C.....SSSS.
.C........S.
.C........S.
.CCCC..SSSS.
............
